src/gui/FxMixerView.cpp

```diff
@@ -185,6 +185,8 @@ void FxMixerView::addNewChannel()
 	updateFxLine(newChannelIndex);
```

Comment:
Well, this method was not written by me so I don't really know.

I am using the "line" terminology on the parts I wrote to keep consistency with all the other parts of the code that refer to the same functionality.